pkgs/by-name/bi/bitwarden-desktop/darwin.nix

```diff
@@ -0,0 +1,25 @@
+{ pname, meta, fetchurl, undmg, stdenv, ... }:
+
+let version = "2024.3.2";
```

Comment:
Not sure what the convention is around let bindings or recursive attests for things like this.